packages/react-components/recipes/src/recipes/media-object/MediaObject.stories.mdx

```diff
@@ -220,7 +220,7 @@ There are also vertical alignment variations, these include start and center as
   <TextAlignmentVariations />
 </TemplateExample>
 
-## **Best practices**
+## **Best Practices**
```

Comment:
Do we need both `##` and `**` at the same time?

Reply:
Weirdly, that seems to be consistently used in that specific file. No idea why, I'll remove it for the level 2 headings at least.